poonam/JS/Assignment8_js/que3.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Choose method for sorting</title>
+    </head>
+    <body>
+        <script>
+            var arrMarks = [];
+            for (var i = 0; i < 10; i++) {
+                marks = parseInt(prompt("Enter marks out of 80 for subject" + (i + 1) + " :"));
+                if (marks > 80) {
+                    alert("Enter marks less than or equals to 80.");
+                }
+                else {
+                    marks = marks + (marks / 10 * 2);
+                    arrMarks.push(marks);
+                }
+                document.write("<br> Marks of subject " + (i + 1) + ": " + arrMarks[i]);
+            }
+            let select = parseInt(prompt("1 – Bubble sort (Press 1)\n2 – Insertion sort (Press 2)\n3 – selection sort(Press 3)\n4 – Quick Sort(Press 5)\n5 – Exit (Press X or Cancel or 5)"));
+            switch (select) {
+                case 1:
+                    bubblesort();
```

Comment:
the array needs to be passed as parameter to all the sorting algo functions.
bubblesort(arrMarks);